benchmark/single-source/ArrayOfGenericPOD.swift

```diff
@@ -40,7 +40,7 @@ class RefArray<T> {
 // elements should be a nop.
 @inline(never)
 func genEnumArray() {
-  _ = RefArray<Int?>(3)
+  blackHole(RefArray<Int?>(3))
   // should be a nop
```

Comment:
I'm confused about all these benchmarks that have the `// should be a nop` comments. What are they supposed to measure?

Reply:
I _think_ it means that it's expected that the optimizer removes everything. I'm not sure if this makes sense.

Reply:
In general, it's better to replace such benchmarks (which only test if the optimizer removes everything) by a lit test.